substrate/frame/benchmarking/src/v1.rs

```diff
@@ -802,13 +803,14 @@ macro_rules! benchmark_backend {
 				$( $param_instancer ; )*
 				$( $post )*
 
-				Ok($crate::__private::Box::new(move || -> Result<(), $crate::BenchmarkError> {
-					$eval;
-					if verify {
-						$postcode;
-					}
-					Ok(())
-				}))
+				recording.start();
+				$eval;
+				recording.stop();
```

Comment:
Not sure how common the name `recording` is, but if someone has a variable with the same name then it could clash, or?  
Maybe  `_recording` or so?

Reply:
Or we add a pair of { ?

Reply:
addressed with: 
https://github.com/paritytech/polkadot-sdk/pull/3934/commits/919759301aac2b47d0b14e40a892275c8db96db4

macro will error if one try to use recording or verify as a variable name

Reply:
okay that works...